### PR TITLE
Added self to mailmap

### PR DESCRIPTION
**Pull request by madphysicist**
Removing duplicate email.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -84,6 +84,7 @@ Jarrod Millman <millman@berkeley.edu> Jarrod Millman <jarrod.millman@gmail.com>
 Jay Bourque <jay.bourque@continuum.io> jayvius <jay.bourque@continuum.io>
 Jerome Kelleher <jerome.kelleher@ed.ac.uk> jeromekelleher <jerome.kelleher@ed.ac.uk>
 Johannes Schönberger <hannesschoenberger@gmail.com> Johannes Schönberger <jschoenberger@demuc.de>
+Joseph Fox-Rabinovitz <jfoxrabinovitz@gmail.com> Joseph Fox-Rabinovitz <joseph.r.fox-rabinovitz@nasa.gov>
```
Comment (charris) [inline .mailmap]: Wrong layout. Goes `Proper Name <proper@email.xx> Commit Name <commit@email.xx>`, where commit name is `madphysicist`.

Comment (madphysicist) [inline .mailmap]: I am trying to get rid of the second line in the current output of `git log --format="%aN <%aE>" | sort -u`:

    Joseph Fox-Rabinovitz <jfoxrabinovitz@gmail.com>
    Joseph Fox-Rabinovitz <joseph.r.fox-rabinovitz@nasa.gov>

I did not notice that there was also this line:

    Mad Physicist <madphysicist@users.noreply.github.com>

Comment (madphysicist) [inline .mailmap]: Fixed now.

**Comment by charris** [comment]
For future reference, if you put `[ci skip]` in the commit message, the tests won't be run. Useful for documentation fixes and other changes that are not tested.

**Pull request merged**